README.md

```diff
@@ -70,6 +70,26 @@ response = client.call(GraphQLRequest(
 model_groups = response["model_groups"]["model_groups"]
 ```
 
+# Testing the SDK
+
+To run the tests associated with this repo perform the following:
+### Prerequisite
+Set the `INDICO_API_TOKEN` environment variable to the token of the environment you need to run tests against.
```

Comment:
```suggestion
Set the `INDICO_HOST` environment variable to the address of the environment you'd like to run the integration tests against, and set `INDICO_API_TOKEN` to the token of the environment you need to run tests against. For more information about how to obtain a token, visit [the documentation](https://docs.indicodata.ai/articles/#!common-questions-publication/how-do-i-get-started-developing-with-the-indico-api/q/API%2520token/qid/3328/qp/1). 
```

Reply:
`INDICO_HOST` is not used in the tests. The tests derive the host from the `--host` cli arg when calling pytest. Made a point here: https://github.com/IndicoDataSolutions/indico-client-python/blob/5a4c85a121b3d0a26a8d4e3550603fa3cac3c540/README.md?plain=1#L90

Reply:
You can actually do either -- if you pass the `--host` argument to `pytest`, it automatically sets the `INDICO_HOST` environment variable: https://github.com/IndicoDataSolutions/indico-client-python/blob/master/tests/integration/conftest.py#L17

I think it's a little confusing that you can set it either way-- it makes most sense to me to remove the `pytest` option and require folks to set the host and the token via env. but *shrugs* it doesn't matter that much for the purposes of this PR.

Reply:
Even if you don't pass `--host` the default setting will cause it to default to `dev.indico.io`. I guess question for @goatrocks, sboudl we just remove this and rely on the environment variable to be available? Or was this potentially necessary for CI reason?